radio/src/gui/212x64/model_setup.cpp

```diff
@@ -853,6 +853,12 @@ void menuModelSetup(event_t event)
           lcdDrawNumber(lcdLastPos, y, moduleData.channelsStart+1, LEFT | (menuHorizontalPosition==0 ? attr : 0));
           lcdDrawChar(lcdLastPos, y, '-');
           lcdDrawNumber(lcdLastPos + FW+1, y, moduleData.channelsStart+NUM_CHANNELS(moduleIdx), LEFT | (menuHorizontalPosition==1 ? attr : 0));
+          if (IS_MODULE_XJT(moduleIdx) && g_model.moduleData[INTERNAL_MODULE].rfProtocol== 0) {
```

Comment:
second INTERNAL_MODULE is wrong, should be moduleIdx as well. and you should use == RF_PROTO_X16 instead of == 0